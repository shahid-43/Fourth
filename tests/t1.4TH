5 3 + . CR .
3 2 + .  