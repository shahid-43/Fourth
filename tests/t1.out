8
5
Execution finished with an empty stack.